games/r.yaml

```diff
@@ -225,6 +225,35 @@
   type: remake
   updated: 2022-11-13
 
+- name: re3-gd "Remake 3 Godot"
+  info: A open-source Grand Theft Auto III reimplementation on the Godot Engine
+  frameworks:
+  - Godot
+  langs:
+  - C++
+  - GDScript
+  licenses:
+  - MIT
+  content: commercial
+  development: very active
+  originals:
+  - Grand Theft Auto III
+  repo: https://github.com/FOSS-Supremacy/re3-gd
+  status: playable
```

Comment:
```suggestion
  status: unplayable
```

Reply:
It's playable but incomplete.

Reply:
according to OSGC's definitions (check osgameclones.com) playable should mean that the main part of the game is fully playable, for example the main campaign is complete in a single player game. Semi-playable also requires some part of the game to be playable, e.g. one mission, or one level. Since this project is only at the stage where assets can be loaded and viewed, it should be unplayable.
https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L46-L48